netman/adapters/switches/juniper/qfx_copper.py

```diff
@@ -42,3 +42,9 @@ def add_enslave_to_bond_operations(self, update, interface, bond):
 
     def add_update_bond_members_speed_operations(self, update, slave_nodes, speed):
         pass
+
+    def get_interface_trunk_native_vlan_id_node(self, interface):
+        return interface.xpath("native-vlan-id")
+
```

Comment:
i'm pretty sure one of the interfaces of the get_interfaces has not native vlan, this will return None and all callers seems to be handling it